### PR TITLE
Fix namespace on the otherConstraints field

### PR DESCRIPTION
**Pull request by markhepburn**

### CODE REVIEW

File: iso19139.mcp-2.0/index-fields.xsl [hunk 1]
```diff
@@ -329,7 +329,7 @@
 				<xsl:for-each select="*/mcp:attributionConstraints/gco:CharacterString">
 					<Field name="attrConstr" string="{string(.)}" store="true" index="true" />
 				</xsl:for-each>
-				<xsl:for-each select="*/gmd:otherConstraints/gco:CharacterString">
+				<xsl:for-each select="*/mcp:otherConstraints/gco:CharacterString">
```
Comment (jonescc) [inline iso19139.mcp-2.0/index-fields.xsl]: I don't think you have to use an mcp constraints element so we should ensure this also works for the old element.  i.e.

   <xsl:for-each select="*/mcp:otherConstraints/gco:CharacterString|*/gmd:otherConstraints/gco:CharacterString">

Comment (markhepburn) [inline iso19139.mcp-2.0/index-fields.xsl]: I see -- so there is a similar element under the gmd namespace?

Want me to push that then, or are you happy to?

Comment (jonescc) [inline iso19139.mcp-2.0/index-fields.xsl]: If you can make the changes that would be great.  Thanks.

Comment (jonescc) [inline iso19139.mcp-2.0/index-fields.xsl]: And yes, I checked I can add a gmd namespace element in an mcp 2.0 record.  mcp is a specialisation of gmd.  either can be used

Comment (markhepburn) [inline iso19139.mcp-2.0/index-fields.xsl]: Aah, I see.  Let me know if you want these squashed as well.

**Comment by jonescc** [comment]
Squashing is good here.

**Pull request merged**